x-pack/legacy/plugins/reporting/server/lib/esqueue/worker.js

```diff
@@ -29,7 +29,12 @@ function getLogger(opts, id, logLevel) {
     const tags = ['worker', logLevel];
 
     if (err) {
-      logger(`${message}: ${err.stack  ? err.stack : err }`, tags);
+      const errString = `${message}: ${err.stack ? err.stack : err}`;
+      logger(errString.substring(0, 1000), tags);
```

Comment:
going to change this so we also show the last 1000 characters of a very long string